src/twisted/internet/defer.py

```diff
@@ -1041,7 +1041,7 @@ def maybeSucceed(result: object) -> None:
         return future
 
     @classmethod
-    def fromFuture(cls, future: Future) -> "Deferred[Any]":
+    def fromFuture(cls, future: Future[_T]) -> "Deferred[_T]":
```

Comment:
It looks like `Future` is not generic on Python 3.7, so this fails.

I guess making it a string is the solution?

Reply:
> I guess making it a string is the solution?

Probably yes.

Reply:
3.7 is no longer supported so should be fine now